Adafruit_DAP.cpp

```diff
@@ -45,6 +45,26 @@ bool Adafruit_DAP::begin(int swclk, int swdio, int nreset, ErrorHandler perr) {
   // return true;
 }
 
+void Adafruit_DAP::targetConnect() {
+  Serial.println("Connecting...");
+  // First disconnect, in case this really is e reconnect
+  if (!dap_disconnect())                      perror_exit(error_message);
+
+  char debuggername[100];
+  if (!dap_get_debugger_info(debuggername))   perror_exit(error_message);
+  Serial.print(debuggername); Serial.print("\n\r");
+
+  if (!dap_connect())                         perror_exit(error_message);
+
+  //Serial.println("Configure transfer...");
+  if (!dap_transfer_configure(0, 128, 128))   perror_exit(error_message);
+  //Serial.println("SWD configure...");
+  if (!dap_swd_configure(0))                  perror_exit(error_message);
+
+  if (!dap_swj_clock(50))                     perror_exit(error_message);
```

Comment:
If there is a need to change some of the parameters, I guess you're right about the clock. It could have a default value of 50 in the class definition. 
The targetConnect method could also be made virtual, so that a user of the library could reimplement it in a subclass, if they need to do something unusual.